lib/graceful-exit.js

```diff
@@ -80,8 +80,10 @@ exports.gracefulExitHandler = function(app, server, _options) {
 
   // Disconnect all the socket.io clients
   if (options.socketio) {
-    options.socketio.sockets.clients().forEach(function(socket) {
-      logger('Killing socketio socket');
+    var connectedSockets = Object.values(options.socketio.sockets.sockets);
+    logger('Disconnecting ' + connectedSockets.length + ' sockets...');
+    options.socketio.close();
+    connectedSockets.forEach(function(socket) {
```

Comment:
Changes are most likely not backward compatible, though I haven't check. The major increase to version `1.0.0` should be good for not affecting other people already using this.